libraries/AP_HAL_ChibiOS/hwdef/common/stm32h7_mcuconf.h

```diff
@@ -241,6 +250,25 @@
 #define STM32_PLL3_DIVN_VALUE               48
 #define STM32_PLL3_DIVQ_VALUE               5
 #define STM32_PLL3_DIVR_VALUE               8
+
+#elif STM32_HSECLK == 32000000U
```

Comment:
this can be folded into the 8 and 16 MHz case, no need for separate handling as you have already divided in the earlier section

Reply:
done